pkg/controller/nodeipam/ipam/multinetwork_cloud_cidr_allocator.go

```diff
@@ -77,7 +102,9 @@ func (ca *cloudCIDRAllocator) performMultiNetworkCIDRAllocation(node *v1.Node, i
 					}
 				} else {
 					northInterfaces = append(northInterfaces, networkv1.NorthInterface{Network: network.Name, IpAddress: inf.NetworkIP})
```

Comment:
should we push northInterfaces also within the `statusNetworks` check below? @sdmodi wdyt?

Reply:
No we should not. Without northInterface populated, anetd will never set network status to ready. The question really was, whether we should do both northInterface and networks annotation at the same time and we decided not to do so.